Framework/Utils/include/Utils/Utils.h

```diff
@@ -25,6 +25,9 @@ namespace o2
 {
 namespace workflows
 {
+//TODO: this is to make DPLmerger compile, but this code is (and always was) horribly broken - please remove.
+inline void freefn(void* data, void* /*hint*/) {delete static_cast<char*>(data);};
```

Comment:
Not sure about your comment. The free-function pointer is required by `FairMQDevice::NewMessageFor`. Agreed, using FreeFn of `Stack` for this purpose is probably awkward, but since you added the function that should be fully valid. now. What do you think should be removed?

Reply:
DPLmerger - remove or fix somehow, that code never had a chance of working - they use a std::vector and just pass the pointer to the first element with the free function. A proper fix (using a std::vector with a pmr allocator) is contained in #1514 of which this PR is a subset - I'll make a new PR that contains the fix. Of course there are other ways of fixing it :)

Reply:
@matthiasrichter the proper fix is in #1593 , depends on this PR

Reply:
very good, thanks